lib/ansible/modules/windows/win_shortcut.ps1

```diff
@@ -45,9 +48,192 @@ if ($null -ne $description) {
     $description = [System.Environment]::ExpandEnvironmentVariables($description)
 }
 
+$module.Result.changed = $false
```

Comment:
So you are adding it again ;-)

Reply:
Fun, I copied your original work before the PR was updated, will remove.